src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
```

Comment:
![image](https://github.com/mate-academy/jv-spring-security-part-2/assets/20326858/ea7e394a-5a39-4570-90c1-4866e637d653)

Reply:
Do a task and not forget something? Unreal! ))